src/javascript/app/base/binary_pages.js

```diff
@@ -148,6 +149,7 @@ const pages_config = {
     'types-of-accounts'      : { module: TypesOfAccounts },
     'video-facility'         : { module: VideoFacility,              is_authenticated: true, only_real: true },
     'why-us'                 : { module: WhyUs },
+    'responsible-trading'    : { module: ResponsibleTrading },
```

Comment:
this should be alphabetically sorted among the single quotation pages, so on line 145